modules/configuring-an-authentication-provider-in-central.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * cli/getting-started-cli.adoc
+:_content-type: PROCEDURE
+[id="configuring-an-authentication-provider-in-central_{context}"]
+= Configuring an authentication provider in Central
+
+[role="_abstract"]
+You can configure an authentication provider in Central and initiate the login process with the `roxctl` CLI.
+Set the variable `ROX _ENDPOINT`, initiate the login process with the `roxctl central login` command, select the authentication provider in a browser window, and retrieve the token information from the `roxctl` CLI.
+
+.Procedure
+
+. Select an authentication provider of your choice, such as OpenID Connect (OIDC) with fragment or query mode, and configure it in Central.
+
+. Run the following command to set the `ROX _ENDPOINT` variable to Central hostname and port:
++
+[source,terminal,subs="+quotes"]
+----
+export ROX_ENDPOINT=_<central_hostname:port>_
+----
+
+. Run the following command to initiate the login process to Central:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl central login
+----
+
+. Within `roxctl` CLI, a URL is printed as output and you are redirected to a browser window where you can select the authentication provider you want to use.
+
+. Once you have successfully logged in with the authentication provider, the browser window indicates that authentication was successful and you can close the browser window.
+
+. `roxctl` CLI prints your token information. This includes details such as the access token, the access token expiration time, the refresh token, and the notification that these values are stored locally.
++
+.Example output
+====
+----
+Please complete the authorization flow in the browser with an auth provider of your choice.
+If no browser window opens, please click on the following URL:
+        http://127.0.0.1:xxxxx/login
+
+INFO:	Received the following after the authorization flow from Central:
+INFO:	Access token: <redacted> <1>
+INFO:	Access token expiration: 2023-04-19 13:58:43 +0000 UTC <2>
+INFO:	Refresh token: <redacted> <3>
+INFO:	Storing these values under $HOME/.roxctl/login… <4>
+----
+====
+<1> The access token.
+<2> The access token expiration time.
+<3> The refresh token.
+<4> The file path where values of the access token, the access token expiration time, and the refresh token are stored locally.
++
+[NOTE]
+====
+Token information is stored locally in one of the following file paths:
```

Comment:
I think I was a bit unclear in the docs input, sorry for that!

The config won't be stored in multiple locations, but rather depending on environment variables set by the user.

It follows the following principle:

If the user sets `$ROX_CONFIG_DIR` variable, then the config will be stored there. This option has the highest priority.

If `$XDG_RUNTIME_DIR` is set, then the config will be stored there, but only if `$ROX_CONFIG_DIR` is unset.

If neither variables are set, the configuration will be stored under `$HOME/.roxctl/roxctl-config`.

Hope that makes it a bit more clear.